lib/blackjack.rb

```diff
@@ -103,9 +103,9 @@ def split_hand(hand)
   end
 
   def dealer_turn
-    unless player_hands.all?(&:blackjack?) || player_hands.all?(&:busted?)
-      dealer_hand.deal(shoe.draw) while dealer_hand.hit? && !dealer_hand.over?
-    end
+    return unless player_hands.all?(&:blackjack?) ||
```

Comment:
I really kinda disagree with Rubocop's style here. The conditional wrapping seemed more clear.